applications/StructuralMechanicsApplication/custom_processes/apply_multi_point_constraints_process.h

```diff
@@ -88,13 +89,13 @@ class ApplyMultipointConstraintsProcess : public Process
         // IMPORTANT : This constructor is not to be used when using this process in the normal KRATOS process_list of python script
         ProcessInfoPointerType info = mr_model_part.pGetProcessInfo();
         if (info->GetValue(MPC_DATA_CONTAINER) == NULL)
```

Comment:
This shouldn't be a `nullptr`?

Reply:
Thank you for the hint Pooyan. I changed it and pushed the commit.